frontend/src/app/pages/summary/summary.module.ts

```diff
@@ -3,10 +3,11 @@ import { CommonModule } from '@angular/common';
 
 import { TransactionsComponent } from '~/app/pages/summary/transactions/transactions.component';
 import { HttpClientModule } from 'node_modules/@angular/common/http';
-import { AgGridModule } from 'ag-grid-angular';
 import { SummaryRoutingModule } from '~/app/pages/summary/summary-routing.module';
 import { SummaryComponent } from './summary.component';
 import { BalanceComponent } from './balance/balance.component';
+import { UtilModule } from '~/app/util/util.module';
+import { MatTooltipModule } from '@angular/material/tooltip';
```

Comment:
Do you realize you won't be able to use it in the native app? AFAIK there are ready-made components like this for Android (requires a refactor anyway) but there is nothing of this sort for iOS.

Reply:
Yes, I realize, we've been talking about this recently :)

Reply:
Meh